tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts

```diff
@@ -100,6 +107,130 @@ describe('E2E Testing for taqueria taquito plugin', () => {
 		}
 	});
 
+	test('Verify that taqueria taquito plugin can transfer amount of tezos using transfer command from one account to another', async () => {
+		try {
+			// Setting up docker container name
+			dockerName = 'local';
+			environment = 'development';
+
+			// 1. Start local sandbox
+			await exec(`taq start sandbox`, { cwd: `./${taqueriaProjectPath}` });
```

Comment:
You don't need to start a sandbox here, one is already getting started in the beforeAll clause above

Reply:
Yeah, I removed it.